t/t1014-mf-priority-dne.t

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+test_description='Test cancelling active jobs with a late user/bank info load'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+	flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
+'
+
+test_expect_success 'check that mf_priority plugin is loaded' '
+	flux jobtap list | grep mf_priority
+'
+
+test_expect_success 'submit job #1 with no user/bank info loaded to plugin' '
+	jobid1=$(flux mini submit hostname)
+'
+
+test_expect_success 'check that job #1 is in state PRIORITY' '
+	test $(flux jobs -no {state} ${jobid1}) = PRIORITY
+'
```

Comment:
There _could_ be an unlikely race here between these two tests. If the first job is submitted, but the `depend` event isn't emitted before the next test runs, then this second test will fail, because the job isn't quite in PRIORITY state yet.

The fix would be to wait for the `depend` event in one of the two tests

```suggestion
test_expect_success 'submit job #1 with no user/bank info loaded to plugin' '
	jobid1=$(flux mini submit --wait-event=depend hostname)
'

test_expect_success 'check that job #1 is in state PRIORITY' '
	test $(flux jobs -no {state} ${jobid1}) = PRIORITY
'
```

This would be a very rare race for this test since `depend` is emitted immediately by the job manager if there are no `dependency-add` events, however, it is perhaps instructive on writing tests that are race condition free (plus if you run a test under valgrind, race conditions tend to be much more likely)